### PR TITLE
EVG-1919 Default to new agent

### PR DESCRIPTION
**Pull request by bsamek**

### CODE REVIEW

File: taskrunner/host_gateway.go [hunk 1]
```diff
@@ -230,7 +230,7 @@ func startAgentOnRemote(settings *evergreen.Settings, hostObj *host.Host, sshOpt
 		fmt.Sprintf("--log_prefix='%s'", filepath.Join(hostObj.Distro.WorkDir, agentFile)),
 	}
 
-	if os.Getenv("NEW_AGENT") != "" {
+	if os.Getenv("NEW_AGENT") != "false" {
```
Comment (tychoish) [inline taskrunner/host_gateway.go]: I would do == "" rather than != ""

**Comment by tychoish** [comment]
lgtm modulo comment

**Comment by bsamek** [comment]
I'm going to leave this open until after we deploy the agent and it looks good. This will make that permanent.

**Comment by tychoish** [comment]
I feel like we can invert the default now (to make the dance of deploying to staging easier, as we are really only interested in testing the new agent)  and then to make it permanent we just delete all the old code, while maintaining the ability to roll back inbetween.

**Pull request merged**